cache/ttl/cache.go

```diff
@@ -115,14 +115,24 @@ func (cache *Cache) Get(key interface{}) (interface{}, bool) {
 }
 
 // Count returns the number of items in the cache
-// (helpful for tracking memory leaks)
 func (cache *Cache) Count() int {
 	cache.mutex.RLock()
 	count := len(cache.items)
 	cache.mutex.RUnlock()
 	return count
 }
 
+// AllKeys returns all keys store in the cache
+func (cache *Cache) AllKeys() []interface{} {
+	var ret []interface{}
```

Comment:
since the size is known ahead, `make( , len(cache.items))`
this makes sure further `append` won't cause re-allocation of slice